Registry/Registry.EM_COMMON

```diff
@@ -1033,9 +1048,9 @@ state   real   pek_adv        ikjftb  scalar      1         -      i0rusdf=(bdy_
 state   real   pep_adv        ikjftb  scalar      1         -      i0rusdf=(bdy_interp:dt) "pep_adv"       "TKE dissipating rate from EEPS"      "m2 s-3"
 
 # MYJ PBL variables; GBM PBL: EXCH_H, EXCH_M
-state    real   EXCH_H          ikj     misc        1         Z     r         "EXCH_H"               "SCALAR EXCHANGE COEFFICIENTS "                "m2 s-1"
-state    real   EXCH_M          ikj     misc        1         Z     r         "EXCH_M"               "EXCHANGE COEFFICIENTS "                       "m2 s-1"
-state    real  CT              ij      misc        1         -      r        "CT"                    "COUNTERGRADIENT TERM"    "K"
+state    real   EXCH_H          ikj     misc        1         Z     r        "EXCH_H"               "SCALAR EXCHANGE COEFFICIENTS "                "m2 s-1"
```

Comment:
these 3 are just white space changes

Reply:
Jimy,
Some of these we tried to clean up a bit. The above examples were trying to line up the columns.